packages/p/protobufjs.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "protobuf.js",
```

```suggestion
  "name": "protobufjs",
```
Name should match the filename, please :)